gremlin-go/README.md

```diff
@@ -65,91 +58,222 @@ import (
 	if err != nil {
 		fmt.Println(err)
 		return
 	}
+	
 	// Grab the first result from all the results in the ResultSet.
 	result := resultSet.All()[0]
+	
 	// Print the first result.
-	fmt.Println(result.AsString())
+	fmt.Println(result.GetString())
 }
 ```
 
 Note: The exact import name as well as the module prefix for `NewDriverRemoteConnection` may change in the future.
 
+You may need to run `go mod tidy` to import the remaining dependencies of the `gremlin-go` driver, after which you should see an updated `go.mod` file:
+
+```
+module gremlin-go-example
+
+go 1.17
+
+require github.com/lyndonb-bq/tinkerpop/gremlin-go v0.0.0-20220131225152-54920637bf94
+
+require (
+	github.com/google/uuid v1.3.0 // indirect
+	github.com/gorilla/websocket v1.4.2 // indirect
+	github.com/nicksnyder/go-i18n/v2 v2.1.2 // indirect
+	golang.org/x/text v0.3.7 // indirect
+)
+```
+As well as a populated `go.sum` file.
+
 # Go Gremlin Language Variant
 
 [Apache TinkerPop™][tk] is a graph computing framework for both graph databases (OLTP) and graph analytic systems
 (OLAP). [Gremlin][gremlin] is the graph traversal language of TinkerPop. It can be described as a functional,
 data-flow language that enables users to succinctly express complex traversals on (or queries of) their application's
 property graph.
 
-Gremlin-Go implements Gremlin within the Go language and can be used on any Go runtime greater than v1.17. Go's syntax 
-has the same constructs as Java including "dot notation" for function chaining (a.b.c), round bracket function arguments
-(a(b,c)), and support for global namespaces (a(b()) vs a(__.b())). One important distinction with Go and Java is that 
-the functions are capitalized, as is required to export functions is Go. As such, anyone familiar with Gremlin-Java 
-will immediately be able to work with Gremlin-Go.
+Gremlin-Go implements Gremlin within the Go language and can be used on any Go runtime greater than v1.17. One important distinction with Go and Java is that 
+the functions are capitalized, as is required to export functions is Go. 
 
 Gremlin-Go is designed to connect to a "server" that is hosting a TinkerPop-enabled graph system. That "server"
 could be [Gremlin Server][gs] or a [remote Gremlin provider][rgp] that exposes protocols by which Gremlin-Go
 can connect.
 
 A typical connection to a server running on "localhost" that supports the Gremlin Server protocol using websockets
 looks like this:
-<!--
-TODO: Add Go code example of connection to server.
--->
+```go
+package main
+
+import (
+	"fmt"
+	"github.com/lyndonb-bq/tinkerpop/gremlin-go/driver"
+)
+
+func main() {
+	// Creating the connection to the server.
+	driverRemoteConnection, err := gremlingo.NewDriverRemoteConnection("localhost", 8182)
+	// Handle error
+	if err != nil {
+		fmt.Println(err)
+		return
+	}
+	// Create an anonymous traversal source with remote
+	g := gremlingo.Traversal_().WithRemote(driverRemoteConnection)
```

Comment:
Note that this probably won't compile since `g` is not used.

Reply:
Right, this is meant as incomplete, it's the step that leads into the next one where you use it.